hsdio.py

```diff
@@ -63,60 +62,70 @@ def load_xml(self, node):
         device settings
         'node': type is ET.Element. tag should be "HSDIO"
         """
+        
+        self.isInitialized = False
```

Comment:
This is the only version to exist in this Instrument but I would still prefer the other naming convention to be applied here for the sake of consistency.

Reply:
due to PXI code that checks this parameter for multiple devices, they all need to have the same name. We can globally change variables to the underscore format in a future PR.